app/Http/Controllers/ClientPortal/PaymentController.php

```diff
@@ -73,7 +73,7 @@ public function show(Request $request, Payment $payment)
      */
     public function process()
     {
-        $gateway = CompanyGateway::find(request()->input('company_gateway_id'));
+        $gateway = CompanyGateway::findOrFail('id', request()->input('company_gateway_id'));
```

Comment:
This broke, it should have been

```
CompanyGateway::findOrFail(request()->input('company_gateway_id'));
```

Reply:
https://github.com/invoiceninja/invoiceninja/pull/4149 🎉